src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static void printList(Task[] list) {
+        System.out.println("     Here are the tasks in your list:");
```

Comment:
Are you trying to have a tabbed space? Maybe you could consider using \t (tab space) instead for better standardisation & easier coding?

Reply:
Similarly in lines 10, 16-19, 26-29, 37, 40, 43